kahuna/package.json

```diff
@@ -49,6 +50,16 @@
             ]
           }
         }
+      },
+      "github:quantumui/titip@397fd324d5b5cc98ee332f876a100a70f69b273e": {
```

Comment:
Ugh, the titip lib only has a master release, which as @theefer pointed out, shouldn't be used as it may not re-produce reliable results.

TC fails w/out this override - could put this override in the [registry repo](https://github.com/jspm/registry/tree/master/package-overrides/github/quantumui) but it feels a bit wrong to use a commit hash version there...